plugin-ratelimit/README.md

```diff
@@ -0,0 +1,13 @@
+# Rate Limit Throttling Plugin
+This plugin allows limiting the rate of contacting a service using a [token
+bucket](https://en.wikipedia.org/wiki/Token_bucket). To create a bucket, create
+a file ending in `.ratelimit` as follows:
+
+    {
+      "capacity": 1000,
+      "delay": 50
+    }
+
+where `capacity` is the maximum number of tokens that be held in the bucket and
+`delay` is the number of milliseconds to generate a new token. The file name
+will be name of the throttled service.
```

Comment:
Maybe move this part about the file name to be beside the sentence where you talk about the file suffix? Also, "The file name will be" sounds like it would be autogenerated; do you mean "The file name must be"?